espnet2/tasks/hubert.py

```diff
@@ -282,8 +283,37 @@ def build_collate_fn(
             pad=args.collate_fn_conf.get("pad", False),
             rand_crop=args.collate_fn_conf.get("rand_crop", True),
             crop_audio=not args.collect_stats,
+            kernel_size=args.collate_fn_conf["kernel_size"],
```

Comment:
Would there be a risk of using previous configs and pretrained checkpoints of previous versions?

Reply:
Would it be straightforward to use the args.encoder_conf here, instead of using the collate_fn_conf? Maybe it would mislead users to set the collate_fn_conf manually.